_protected/app/system/core/assets/cron/96h/BannedCoreCron.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+/**
+ * @title            Birthday Cron Class
+ *
+ * @author           Pierre-Henry Soria <ph7software@gmail.com>, Polyna-Maude R.-Summerside <polynamaude@gmail.com>
+ * @copyright        (c) 2013-2018, Pierre-Henry Soria. All Rights Reserved.
+ * @license          GNU General Public License; See PH7.LICENSE.txt and PH7.COPYRIGHT.txt in the root directory.
+ * @package          PH7 / App / System / Core / Asset / Cron / 24H
+ * @version          1.0
+ */
+namespace PH7;
+
+defined ( 'PH7' ) or exit ( 'Restricted access' );
+
+use Exception;
+use PH7\Framework\Security\Ban\Ban;
+use PH7\Framework\Error\Logger;
+
+class BannedCoreCron extends Cron {
```

Comment:
Nitpick-CI  complaint here (and below), because the coding style doesn't follow PSR-2 :smiley:
For instance, the curly bracket should be on the next line.
If you use PHPStorm, you can reformat it -> File -> Reformat Code (you should have a similar option or plugin with the other IDE/Text editors).